docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1326,6 +1327,11 @@ Create a new image from a container's changes
              "Cmd":[
                      "date"
              ],
+	     "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
If were going with that, then I think we should remove any restriction that the label content contain a `=`.

Internally the code won't be able to use a map since you can have duplicate keys, so it'll just be an array of strings. If it's an array of strings, I don't see any reason why a `=` would be required.
Doing this also supports the term 'label' as well, as it's no longer a key/value pair (it can be used as such, but it's up to the client).